nmostesting/suites/IS1101Test.py

```diff
@@ -13,14 +13,344 @@
+
+            return test.PASS()
+
+        return test.UNCLEAR("Could not find any IS-04 receivers to test")
+
+    # OUTPUTS TESTS
+    def test_03_01(self, test):
+        """
+        Verify that the device supports the concept of Output.
+        """
+        _, response = TestHelper.do_request("GET", self.compat_url + "outputs/")
+
+        if response.status_code == 200:
+            if len(response.json()) != 0:
+                self.outputs.append(response.json()[0])
+            if len(self.outputs) == 0:
+                return test.UNCLEAR("No outputs")
+            return test.PASS()
+        return test.FAIL(response.json())
+
+    def test_03_02(self, test):
+        """
+        Verify that some of the outputs of the device are connected.
+        """
+        if len(self.outputs) == 0:
+            return test.UNCLEAR("No IS11 receivers outputs")
+        for output in self.outputs:
+            _, response = TestHelper.do_request(
+                "GET", self.compat_url + "outputs/" + output + "properties/"
+            )
+            if response.status_code == 200:
+                outputs_properties_json = []
+                outputs_properties_json.append(response.json())
+                for output in outputs_properties_json:
+                    if output["connected"]:
+                        self.connected_outputs.append(output["id"])
+            else:
+                return test.FAIL(response.json())
+        if len(self.connected_outputs) == 0:
+            return test.UNCLEAR("None Outputs support edid.")
+        return test.PASS()
+
+    def test_03_03(self, test):
+        """
+        Verify that all connected outputs do not have
+        a signal as test 0 put all of the receivers inactive.
+        """
+        if len(self.connected_outputs) == 0:
+            return test.UNCLEAR("no connected outputs")
+        for output_id in self.connected_outputs:
+            _, response = TestHelper.do_request(
+                "GET", self.compat_url + "outputs/" + output_id + "/properties/"
+            )
+            if response.status_code == 200:
+                if response.json()["status"]["state"] == "signal_present":
+                    self.active_connected_outputs.append(response.json())
+            else:
+                return test.FAIL(response.json())
+        if len(self.active_connected_outputs) != 0:
+            return test.UNCLEAR(
+                "Connected output have a signal while all receivers are inactive"
+            )
+        return test.PASS()
+
+    def test_03_04(self, test):
+        """
+        Verify that connected outputs supporting EDID behave according to the RAML file.
+        """
+        if len(self.connected_outputs) == 0:
+            return test.UNCLEAR("no connected outputs")
+        for output_id in self.connected_outputs:
+            _, response = TestHelper.do_request(
+                "GET", self.compat_url + "outputs/" + output_id + "/properties/"
```

Comment:
This just repeats the requests already made in `test_03_02`. Why not just filter the already retrieved `self.connected_outputs`?

Reply:
Is this comment tracked by an issue? Similar question about several test cases.

Reply:
Track by https://github.com/AMWA-TV/nmos-testing/issues/797